src/View/Helper/FormHelper.php

```diff
@@ -1441,10 +1443,28 @@ protected function _magicOptions($fieldName, $options, $allowOverride)
     {
         $context = $this->_getContext();
 
+        $options += [
+            'templateVars' => []
+        ];
+
         if (!isset($options['required']) && $options['type'] !== 'hidden') {
             $options['required'] = $context->isRequired($fieldName);
         }
 
+        if (method_exists($context, 'getRequiredMessage')) {
+            $message = $context->getRequiredMessage($fieldName);
+            $message = addslashes(h($message));
```

Comment:
Why do you need `addslashes()` shouldn't HTML encoding encode all the quote characters?

Reply:
I think that was leftover from a previous version. Removed 👍